### PR TITLE
Dimensions are not respected in tag method

### PR DESCRIPTION
**Pull request by tomgross**
The width and height set in the *tag*-method of the *scaling*-view are not respected.

### CODE REVIEW

File: plone/namedfile/scaling.py [hunk 1]
```diff
@@ -326,4 +326,8 @@ def tag(self,
             direction='thumbnail',
             **kwargs):
         scale = self.scale(fieldname, scale, height, width, direction)
+        if height is not None:
+            kwargs['height'] = height
+        if width is not None:
+            kwargs['width'] = width
```
Comment (davisagli) [inline plone/namedfile/scaling.py]: This doesn't look right to me. The generated scale fits within the requested dimensions, but does not necessarily have those exact dimensions if the image has a different aspect ratio than the requested dimensions.

**Comment by tomgross** [comment]
This fix is motivated by fixing a [failing test of plone.outputfilters](http://jenkins.plone.org/job/plip-plone-app-testing/10/testReport/junit/plone.outputfilters.tests.test_resolveuid_and_caption/ResolveUIDAndCaptionFilterIntegrationTestCase/test_image_captioning_handles_unquoted_attributes/) using p.a.contenttypes.
Now there is a different tag-generation (scaling?) story for AT and DX content. I don't know, if this patch is "right" but it unifies a behavior (which is probably broken already?) between AT and DX content and fixes two existing [tests](http://jenkins.plone.org/job/plip-plone-app-testing/10/testReport/plone.outputfilters.tests.test_resolveuid_and_caption/ResolveUIDAndCaptionFilterIntegrationTestCase/test_image_captioning_preserves_custom_attributes/) when changing from AT content to p.a.contenttypes.
The other option would of course be to change the test assumption in p.outputfilters.

File: plone/namedfile/scaling.py (hunk 1 above)
Comment (jensens) [inline plone/namedfile/scaling.py]: I think here its not the place to handle aspect ratio. This here is low-level, the caller has to care about that. in fact taking care here would i.e. probably break plone.app.imagecropping 
@frisi what do you think?

**Comment by jensens** [comment]
i think we can merge it. if height and width are explicitly given they are set. if this makes problems in addons we can solve it there or prepare a new PR here, this one must not block anything.

**Comment by davisagli** [comment]
I think the correct solution would be to set the tag's height and width to the dimensions of the scale that is generated, not to the height and width that were passed into the tag function.

**Comment by davisagli** [comment]
I put in place a different fix in plone.outputfilters so we don't have to change this. https://github.com/plone/plone.outputfilters/commit/d56c3124a4358410b8a5d850f7fd6d071cb49c95

**Pull request closed without merging**